### PR TITLE
Mithril

### PR DESCRIPTION
**Pull request by jonathanarde**
anything regarding mithril

### CODE REVIEW

File: public/javascripts/export1Table.js [hunk 1]
```diff
@@ -1,133 +1,149 @@
...
-    })
-    return lang
-}
+        return lang
+    }
+
+    var langOptions = langLabelArr(translations2d)
 
-var langOptions = langLabelArr(translations2d)
+<<<<<<< HEAD
```
Comment (chrislim) [inline public/javascripts/export1Table.js]: you have a merge conflict to fix here.

File: public/javascripts/export1Table.js [hunk 2]
```diff
@@ -1,133 +1,149 @@
...
+                        ]),
+                        songs.map((s, i) => {
+                            return m('tr', [
+                                m('td', [
+                                    m('a', {
+                                        href: '/songlist/' + s._id
+                                    }, s.title)
+                                ]),
+                                langOptions.map((l) => {
+                                    return _.includes(translations2d[i].map((t) => t.lang), l) ? m('td', [
```
Comment (chrislim) [inline public/javascripts/export1Table.js]: i recommend moving this to its own function for code readability.

File: public/javascripts/functionLibrary.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ var addSong = function(id) {
         }
     }).then(function(res) {
         res.url ? window.location.href = res.url : inLibrary(res.inLibrary)
+        console.log(res.inLibrary)
```
Comment (chrislim) [inline public/javascripts/functionLibrary.js]: still recommend renaming this file name to be more descriptive. function is a generic javascript concept. What is actually in this library? util.js is generic too, but it indicates this is like a kitchen sink. Unless you mean for all functions to go in this file.

File: public/javascripts/searchBox.js [hunk 4]
```diff
@@ -1,38 +1,45 @@
...
-                    if (!isInit) {
-                        vm.enter(elem);
-                    }
-                }
-            }),
-            m('span.input-group-btn', [
-                m('button#search-button.btn.btn-success', {
-                    onclick: function() {
-                        window.location.href = '/songlist/search?q=' + vm.tag()
+<<<<<<< HEAD
```
Comment (chrislim) [inline public/javascripts/searchBox.js]: merge conflicts.

File: gulpfile.js [hunk 5]
```diff
@@ -3,29 +3,19 @@ var gulp = require('gulp')
 var jshint = require('gulp-jshint')
 var babel = require('gulp-babel');
 
-gulp.task('lint', function() {
```
Comment (chrislim) [inline gulpfile.js]: what do you use to run your server? probably include it as part of your default task.

File: public/javascripts/export1Table.js (hunk 1 above)
Comment (jonathanarde) [inline public/javascripts/export1Table.js]: ok

File: public/javascripts/export1Table.js (hunk 2 above)
Comment (jonathanarde) [inline public/javascripts/export1Table.js]: ok

File: public/javascripts/functionLibrary.js (hunk 3 above)
Comment (jonathanarde) [inline public/javascripts/functionLibrary.js]: i am not using this file right now

**Pull request merged**